packages/http/src/mocker/serializer/style/delimited.ts

```diff
@@ -0,0 +1,50 @@
+import { HttpParamStyles } from '@stoplight/types';
+import * as Either from 'fp-ts/lib/Either';
```

Comment:
I do not really know if the serialiser should be in the http package; this is not going to be used here so I think maybe it makes sense to leave it in the CLI?

Reply:
I got two args for leaving it here:
- it fits by domain here
- I feel that at some point we will create a lib with those serializers / deserializers, so it will be easier to extract (a folk doing it might not even know that there is something in cli package)

I'd still vote on leaving it here.

Reply:
Yes effectively I think you made the good choice. Good call 👍